<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RoomWidget</class>
 <widget class="QWidget" name="RoomWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>587</width>
    <height>405</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QListWidget" name="lwUser">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>40</y>
     <width>121</width>
     <height>191</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>121</width>
     <height>16777215</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="lbVideo">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>381</width>
     <height>201</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(85, 255, 127);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbScroll">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>381</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(85, 255, 127);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>20</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 170, 255);
font: 12pt &quot;04b_21&quot;;</string>
   </property>
   <property name="text">
    <string>用户列表</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="teShow">
   <property name="geometry">
    <rect>
     <x>19</x>
     <y>241</y>
     <width>519</width>
     <height>101</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>101</height>
    </size>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>350</y>
     <width>521</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLineEdit" name="leIput"/>
    </item>
    <item>
     <widget class="QPushButton" name="btnSend">
      <property name="minimumSize">
       <size>
        <width>75</width>
        <height>23</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>75</width>
        <height>23</height>
       </size>
      </property>
      <property name="text">
       <string>发送</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton">
      <property name="minimumSize">
       <size>
        <width>75</width>
        <height>23</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>75</width>
        <height>23</height>
       </size>
      </property>
      <property name="text">
       <string>送礼物</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QCheckBox" name="cbCamera">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>10</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>摄像头</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>10</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>弹幕开关</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
